### PR TITLE
height width required for next_image

### PR DESCRIPTION
**Pull request by Lendemor**
`width` and `height` are now required
```python
rx.image(src="/favicon.ico", width=150, height=100) 
```

### CODE REVIEW

**Review by masenf — APPROVED** [review]
we can either take this as an interim, or leave it open to add the fallback logic discussed in the sync up today.

File: reflex/components/next/image.py [hunk 1]
```diff
@@ -80,6 +83,27 @@ def create(
         Returns:
             _type_: _description_
         """
+        if "width" not in props or "height" not in props:
+            # props.pop("width", None)
+            # props.pop("height", None)
+            if "width" in props:
+                props["width"] = f"{props['width']}px"
```
Comment (picklelo) [inline reflex/components/next/image.py:90]: We should only do this if they're not already setting a unit

File: reflex/components/next/image.py [hunk 2]
```diff
@@ -80,6 +83,27 @@ def create(
...
+            if "height" in props:
+                props["height"] = f"{props['height']}px"
+                # return Box.create(
```
Comment (picklelo) [inline reflex/components/next/image.py:93]: What's the issue in using a Box?

File: reflex/components/next/image.py (hunk 1 above)
Comment (Lendemor) [reply reflex/components/next/image.py]: next_image only accept int anyway so they can't set unit

File: reflex/components/next/image.py (hunk 2 above)
Comment (Lendemor) [reply reflex/components/next/image.py]: We've have to know both width and height if we want the box to constrain an image set to 100%, it won't work.

**Review by masenf — COMMENTED** [review]
Do we still want/need this? I know next/image isn't the default `rx.image` anymore, but is it usable enough through `rx.next.image` to include? Is next/image even desirable for us, given that it does not work with static exports?

**Pull request closed without merging**